### PR TITLE
Add a known issues doc.

### PR DESCRIPTION
**Pull request by brendandburns**
@thockin @RichieEscarez @bgrant0607 @dchen1107 @saad-ali

### CODE REVIEW

**Comment by brendandburns** [comment]
Closes https://github.com/GoogleCloudPlatform/kubernetes/issues/10760

File: docs/user-guide/known-issues.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+This document summarizes known issues with existing Kubernetes releases.
+
+Please consult this document before filing new bugs.
+
+### Release 1.0.1
+
+   * If multiple Pods use the same RBD volume in read-write mode, it is possible data on the RBD volume could get corrupted. This problem has been found in environments where both apiserver and etcd rebooted and Pod were redistributed.
+      * A workaround is to ensure there is no other Ceph client uses the RBD volume before mapping RBD image in read-write mode. For example, rados -p poolname listwatchers image_name.rbd can list RBD clients that are mapping the image.
+   * `exec` liveness/readiness probes leak resources due to Docker exec leaking resources (#10659)
+   * `docker load` sometimes hangs which causes the `kube-apiserver not to start.  Restarting the Docker daemon should fix the issue (#10868)
```
Comment (ixdy) [inline docs/user-guide/known-issues.md]: missing ` after kube-apiserver

Comment (brendandburns) [inline docs/user-guide/known-issues.md]: fixed.

**Comment by brendandburns** [comment]
Comment addressed, please re-check.

Thanks
--brendan

File: docs/user-guide/known-issues.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## Known Issues
+
+This document summarizes known issues with existing Kubernetes releases.
+
+Please consult this document before filing new bugs.
+
+### Release 1.0.1
+
+   * If multiple Pods use the same RBD volume in read-write mode, it is possible data on the RBD volume could get corrupted. This problem has been found in environments where both apiserver and etcd rebooted and Pod were redistributed.
```
Comment (davidopp) [inline docs/user-guide/known-issues.md]: Pod were redistributed -> Pods were redistributed

File: docs/user-guide/known-issues.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      * A workaround is to ensure there is no other Ceph client uses the RBD volume before mapping RBD image in read-write mode. For example, rados -p poolname listwatchers image_name.rbd can list RBD clients that are mapping the image.
```
Comment (davidopp) [inline docs/user-guide/known-issues.md]: client uses -> client using

**Comment by davidopp** [comment]
To save @brendandburns some time I re-created this PR with the reviewer comments applied, see #11627.

**Pull request closed without merging**